app/assets/javascripts/events.js.coffee

```diff
@@ -93,3 +93,15 @@ $ ->
           element: @
         )
         map.map.addOverlay(overlay)
+
+  if $('.toggle-past-events').length
+    $('.toggle-past-events').click (evt) =>
+      pastEvents = $('.past-events')
+      spanTxt = $('.toggle-past-events').children('span')
+
+      if pastEvents.hasClass('hidden')
+        pastEvents.removeClass('hidden')
+        spanTxt.text('Hide')
```

Comment:
Yes, thanks. I think that in coffeescript will be:

```javascript
if pastEvents.hasClass('hidden') then spanTxt.text('Hide') else spanTxt.text('Show')
```
http://coffeescript.org/#expressions